pkg/packagekit/updates.jsx

```diff
@@ -255,34 +253,23 @@ class UpdateItem extends React.Component {
                 errata = null; // simpler testing below
         }
 
+        var secSeverity = getSeverityURL(info.vendor_urls);
+        var iconClasses = PK.getSeverityIcon(info.severity, secSeverity);
         var type;
-        var secSeverity;
         if (info.severity === PK.Enum.INFO_SECURITY) {
-            let classes = "pficon pficon-security";
-
-            // parse Red Hat security update classification from vendor_urls
-            secSeverity = getSeverityURL(info.vendor_urls);
             if (secSeverity) {
                 let s = secSeverity.slice(secSeverity.indexOf("#") + 1);
```

Comment:
You're doing this twice now (here and in the new `getSeverityIcon()`). Could you please do this once after line 257 above, use that value here and then pass the severity itself into `getSeverityIcon()` instead of the full URL?

Reply:
Ack, good point. Rearranged the code accordingly.